guess_the_number.js

```diff
@@ -0,0 +1,41 @@
+import rl from 'readline-sync'
+
+const theTruth = Math.floor( Math.random() * 100 ) + 1
+
+const isValid = input => {
+  return  ( parseInt( input ) === true ) ||
+          ( parseInt(input) < 101 && parseInt(input) > 0 )
+}
+
+const getInput = () => {
+  let guess
+  do {
+      guess = rl.question( 'A number between 1 and 100 please: ' )
+      if ( guess === 'exit' )
+        return guess
+  } while( !isValid( guess ) )
+
+  return parseInt(guess)
+}
+
+const checkForWin = ( userGuess ) => {
+  if ( userGuess === theTruth ) {
+    console.log('You Win!')
+  } else {
+      let difference = theTruth - userGuess
+      if ( difference < 0 ) {
+        console.log( 'You guessed too high!' )
+      } else {
+        console.log( 'Your guess was too low!' )
+      }
+  }
+}
+
+const numberGame = () => {
+  let userGuess = getInput()
+  if ( userGuess === 'exit' )
```

Comment:
Two notes on this line: 
1. Use braces to make the block explicit, as discussed in a previous comment
2. Since the string `exit` is used multiple times, I would refactor this into a constant that can be used throughout the code.  This avoids any inadvertent bugs that could be introduced via typo:
```js
const EXIT_GUESS = 'exit'
```